warehouse/query-core/src/main/java/datawave/query/iterator/QueryInformationIterator.java

```diff
@@ -31,7 +31,7 @@
 public class QueryInformationIterator extends WrappingIterator {
     private static final Logger log = Logger.getLogger(QueryInformationIterator.class);
     
-    private static final String REPORT_ERRORS_OPT = "REPORT_ERRORS";
+    protected static final String REPORT_ERRORS_OPT = "REPORT_ERRORS";
```

Comment:
maybe public?

Reply:
This was just part of the larger refactoring of the ErrorReportingIterator (subclass), which was masking protected fields here and also duplicating a lot of code needlessly.

The spotbugs findings were....
```
[ERROR] Field ErrorReportingIterator.errorKey masks field in superclass datawave.query.iterator.QueryInformationIterator [datawave.query.iterator.errors.ErrorReportingIterator] In ErrorReportingIterator.java MF_CLASS_MASKS_FIELD
[ERROR] Field ErrorReportingIterator.returned masks field in superclass datawave.query.iterator.QueryInformationIterator [datawave.query.iterator.errors.ErrorReportingIterator] In ErrorReportingIterator.java MF_CLASS_MASKS_FIELD
```
...so I also used the opportunity to get rid of ErrorReportingIterator's duplicated methods, and also to delete it's duplicate private `REPORT_ERRORS_OPT`, which is why I made this one protected here in the superclass. Figured it was better to keep the visibility limited